doc/api/n-api.md

```diff
@@ -3420,6 +3420,88 @@ This API returns the names of the enumerable properties of `object` as an array
 of strings. The properties of `object` whose key is a symbol will not be
 included.
 
+#### napi_key_collection_mode
```

Comment:
If types were expected to be hoisted to the top of the document?

Reply:
@Himself65 there is a section named `## Basic N-API Data Types` *almost* at the top of the file. Could you please move these to the bottom of that section?